### DISCUSSION TITLE
[7주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#55] [General]
### 진행순서
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [Java] Garbage Collection #14 
2. 이주형
 * [Java] Exception 종류와 처리 방법 #52
3. 김정윤
 * [ETC] CI/CD란 #54
4. 이동호
 * [Java] DTO와 VO #56

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😊
* 좋았던 점
* 건의사항
* 느낀점

**Reply by doho-ho** [reply]
감사합니다

**Reply by KJY97** [reply]
GC를 너무 단순하게 생각했나봐요.. 깊게 들어가니 어렵네요ㅠ
감사합니다~~

**Reply by ChaerinYu** [reply]
2주 만에 스터디하니 힘드네요.. ㅠ 공부도 꾸준히 .. .. ...

**Reply by leeejuhyeong** [reply]
고생하셨씁니다
